docs/troubleshooting/troubleshooting.md

```diff
@@ -89,6 +89,24 @@ Check connectivity between AKO Pod and Avi controller.
 
   Refer the [document](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.25/#securitycontext-v1-core) to select the best suitable configuration.
 
+#### Static routes not getting populated in vrfcontext
```

Comment:
We are mentioning this for HA use case so we should add it `When AKO is running in HA, ....` to clarify user when to use this.

Reply:
As per the discussion with Anurag in the meeting, he has asked not to specifically say that this should be done in this case. Instead, just say something like whenever static routes are not getting populated reboot all the ako pods both in HA and no HA mode